tools/add_triage_signature.py

```diff
@@ -839,6 +839,56 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket(report)
 
 
+class DownloadImage(ErrorSignature):
+    image_download_regex = re.compile(r"Host (.+?): New image status (.+?). result:.+?; download rate: (.+?) MBps")
```

Comment:
This is "download + store on disk" rate effectively, right?

Reply:
I don't think the target file system for image pulls during discovery is disk bound

Reply:
Might be wrong though

Reply:
And if I'm wrong we should probably change it to be RAM backed

Reply:
Right, we do `podman pull` which includes "write on disk" time, but since it all happens on LiveCD we're writing into tmpfs iiuc, not "store on target disk"